src/java.base/share/classes/com/sun/crypto/provider/AESKeyWrap.java

```diff
@@ -112,20 +112,20 @@ void init(boolean decrypting, String algorithm, byte[] key, byte[] iv)
     @Override
     void reset() {
         throw new UnsupportedOperationException("reset not supported");
-    };
+    }
 
 
-    // no support for multi-part encryption
+    // no support for multipart encryption
     @Override
     int encrypt(byte[] pt, int ptOfs, int ptLen, byte[] ct, int ctOfs) {
         throw new UnsupportedOperationException("multi-part not supported");
```

Comment:
Change the error message as well?

Reply:
Yes. None of our tests check for this error message.